test/scheduleTest.ts

```diff
@@ -24,6 +24,87 @@ describe('schedule', function () {
     assert(job)
   })
 
+  it('should send jobs based on every one second expression', async function () {
+    const config = {
+      ...this.bossConfig,
+      cronMonitorIntervalSeconds: 1,
```

Comment:
can foresee github issues being created when someone adds a fast cron (e.g. every five seconds) but doesn't set `cronMonitorIntervalSeconds` to be low enough. We can document this but wondering if there's anything else we should do?